src/backend/InvenTree/order/models.py

```diff
@@ -1951,6 +1951,7 @@ class SalesOrderShipmentReportContext(report.mixins.BaseReportContext):
 
 class SalesOrderShipment(
     InvenTree.models.InvenTreeAttachmentMixin,
+    InvenTree.models.InvenTreeBarcodeMixin,
```

Comment:
If you add the BarcodeMixin, you also need to implement the `barcode_model_type_code`.

E.g.

https://github.com/inventree/InvenTree/blob/3d3a1b01c73e32707cff7779a0e54803272dbaad/src/backend/InvenTree/company/models.py#L684-L687